### PR TITLE
[DOCU-2183] Gateway 2.8 changelog

### PR DESCRIPTION
**Pull request by lena-larionova**
### Summary
Changelog for 2.8.

Compiled using:

- Enterprise changelog: https://konghq.atlassian.net/wiki/spaces/FTT/pages/2720333958/Kong+Enterprise+2.8.0.0+Next+Minor
- OSS changelog: https://github.com/Kong/kong/blob/master/CHANGELOG.md
- PRs in the Gateway 2.8 milestone: https://github.com/Kong/docs.konghq.com/milestone/14
- Tickets tagged with 2.8.0.0

Note that not all links are live. Secrets management and the /developers endpoint have not yet been merged.

### Reason
2.8 release

### Testing
[Changelog preview](https://deploy-preview-3713--kongdocs.netlify.app/gateway/changelog)

### CODE REVIEW

**Review by PaulFischer-Kong — APPROVED** [review]
LGTM. 
Will need to update the "beta feature" text once secret mgmt is GA.

**Comment by lena-larionova** [comment]
@PaulFischer-Kong need your eyes on one more thing to get the phrasing correct: https://github.com/Kong/docs.konghq.com/pull/3713/files#diff-c2d9c755d22ea9844e88d58c0184e3c5d63133ceb82c327596483c7ad20c672dR468-R492

Want to make sure we say the correct thing about CentOS deprecation.

On a related note, please also look over https://github.com/Kong/docs.konghq.com/pull/3729, which is the PR with the non-changelog updates for the same thing.

File: app/gateway/changelog.md [hunk 1]
```diff
@@ -3,6 +3,493 @@ title: Kong Gateway Changelog
...
+  * Added three new metrics:
+    * `kong_db_entities_total` (gauge): total number of entities in the database.
+    * `kong_db_entity_count_errors` (counter): measures the number of errors encountered during the measurement of `kong_db_entities_total`.
+    * `kong_nginx_timers` (gauge): total number of Nginx timers, in Running or Pending state. Tracks `ngx.timer.running_count()` and
+      `ngx.timer.pending_count()`.
+      [#8387](https://github.com/Kong/kong/pull/8387)
+
+* [OpenID Connect](/hub/kong-inc/openid-connect/) (`openid-connect`)
+
+  * Add Redis ACL support (Redis v6.0.0+) for storing and retrieving a session.
```
Comment (GayleNeumann) [inline app/gateway/changelog.md:124]: `  * Added Redis ACL support (Redis v6.0.0+) for storing and retrieving a session.`

File: app/gateway/changelog.md [hunk 2]
```diff
@@ -3,6 +3,493 @@ title: Kong Gateway Changelog
...
+  the dot is not part of the hostname.
+  [#8269](https://github.com/Kong/kong/pull/8269)
+
+* Fixed a bug in the Router that would not prioritize the routes with
+  both a wildcard and a port (`route.*:80`) over wildcard-only routes (`route.*`),
+  which have less specificity.
+  [#8233](https://github.com/Kong/kong/pull/8233)
+
+* The internal DNS client isn't confused by the single-dot (`.`) domain,
+  which can appear in `/etc/resolv.conf` in special cases like `search .`
```
Comment (GayleNeumann) [inline app/gateway/changelog.md:317]: If possible, remove the space between search and the period: `search.`

File: app/gateway/changelog.md [hunk 3]
```diff
@@ -3,6 +3,493 @@ title: Kong Gateway Changelog
...
+
+#### Clustering
+
+* Replaced a cryptic error message with a more useful one when
+  there is a failure on SSL when connecting with the control plane.
+  [#8260](https://github.com/Kong/kong/pull/8260)
+
+#### Admin API
+
+* Fix an incorrect `next` field that appeared when paginating Upstreams.
```
Comment (GayleNeumann) [inline app/gateway/changelog.md:342]: `* Fixed an incorrect `next` field that appeared when paginating Upstreams.`

File: app/gateway/changelog.md [hunk 4]
```diff
@@ -3,6 +3,493 @@ title: Kong Gateway Changelog
...
+    [#8401](https://github.com/Kong/kong/pull/8401)
+
+* [AWS Lambda](/hub/kong-inc/aws-lambda/) (`aws-lambda`)
+  * Fixed incorrect behavior when configured to use an HTTP proxy
+  and deprecated the `proxy_scheme` config attribute for removal in 3.0.
+  [#8406](https://github.com/Kong/kong/pull/8406)
+
+* [OAuth2](/hub/kong-inc/oauth2/) (`oauth2`)
+  * The plugin clears the `X-Authenticated-UserId` and
+  `X-Authenticated-Scope` headers when it configured in logical OR and
```
Comment (GayleNeumann) [inline app/gateway/changelog.md:384]: `  `X-Authenticated-Scope` headers when it is configured in logical OR and`

File: app/gateway/changelog.md (hunk 2 above)
Comment (lena-larionova) [reply app/gateway/changelog.md:317]: This is not a period. This is referring to the single-dot (`.`) domain, so `search .` means search everything in current/root location.

**Review by GayleNeumann — APPROVED** [review]
LGTM -- some minor edits, but otherwise great work on this very large changelog!!

**Pull request merged**